src/grpc/clients/ea_restaurant_java_etl_grpc_client.py

```diff
@@ -1,13 +1,21 @@
+from injector import Module, inject
+
+
+from src.constants.grpc_status import StatusCode
+from src.env_config import get_env_config_instance
+from src.grpc.clients.grpc_client import GrpcClient
 from src.proto import java_etl_grpc_client_pb2
 from src.utils.grpc_util import (
     generate_basic_token_from_credentials,
     map_mongo_order_status_histories_to_grpc_mongo_order_status_histories,
+    generate_bearer_token_from_access_token,
 )
 from src.proto import java_etl_grpc_client_pb2_grpc
 
 
-class EaRestaurantJavaEtlGrpcClient:
-    def __init__(self, grpc_client):
+class EaRestaurantJavaEtlGrpcClient(Module):
+    @inject
+    def __init__(self, grpc_client: GrpcClient):
```

Comment:
Well done!!!

I think you can also send client_id and client_secret.

Reply:
Done.